packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -316,6 +316,14 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
   }
 
   public componentWillUnmount(): void {
+    this._events.off(window, 'resize', this._onAsyncResize);
```

Comment:
Doesn't disposing the EventGroup automatically do this for each registered listener?